Source/WebCore/platform/graphics/TextRun.cpp

```diff
@@ -34,7 +34,8 @@ struct ExpectedTextRunSize {
     float float1;
     float float2;
     float float3;
-    unsigned bitfields : 9;
+    ExpansionBehavior expansionBehavior;
+    unsigned bitfields : 5;
 };
 
 static_assert(sizeof(TextRun) == sizeof(ExpectedTextRunSize), "TextRun is not of expected size");
```

Comment:
I know you didn't write this code, but it's probably worth changing the error message to say "TextRun should be small"